Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetVerticalUITest.swift

```diff
@@ -30,7 +30,7 @@ class PaymentSheetVerticalUITests: PaymentSheetUITestCase {
         settings.mode = .setup
         settings.customerMode = .new
         settings.currency = .eur
-        settings.customerKeyType = .legacy // TODO: Change to customerSessions when adding save checkbox for SEPA
```

Comment:
I also see a TODO like this in PaymentSheetStandardLPMUIOneTests.testSEPADebitPaymentMethod_PaymentSheet. Should we update that one too?

Reply:
YES! thank you!!